luarules/gadgets/scav_spawner_defense.lua

```diff
@@ -156,12 +156,17 @@ if gadgetHandler:IsSyncedCode() then
 	HumanTechLevel = 0
 
 	--dynamic difficulty stuff
-	local dynDifficulty0to1 -- ranges from 0-1 based on if the scavengers last wave compared against all non-scav/raptor team powers. 1/5th = 0, 1/3rd = 1 with a linear progression between
-	local dynDifficultyClamped -- ranges from .85 to 1.05 based on dynDifficulty0to1 with a linear progression in between.
-	
+	local dynamicDifficulty
+	local dynamicDifficultyClamped
 	local peakScavPower
 	local totalPlayerTeamPower
-
+	
+	--config calculateDifficultyMultiplier
+	local ratio = peakScavPower / totalPlayerTeamPower
```

Comment:
`ratio` is dynamic, not static, it needs to be declared inside the function as the value will be different each time it is called.